# --- T2-COPYRIGHT-NOTE-BEGIN ---
# T2 SDE: package/*/qt5base/hotfix-gcc11.patch
# Copyright (C) 2021 - 2022 The T2 SDE Project
# 
# This Copyright note is generated by scripts/Create-CopyPatch,
# more information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License version 2 as used by the T2 SDE.
# --- T2-COPYRIGHT-NOTE-END ---

--- qtbase/src/corelib/global/qendian.h
+++ qtbase/src/corelib/global/qendian.h
@@ -44,6 +44,8 @@
 #include <QtCore/qfloat16.h>
 #include <QtCore/qglobal.h>
 
+#include <limits>
+
 // include stdlib.h and hope that it defines __GLIBC__ for glibc-based systems
 #include <stdlib.h>
 #include <string.h>
--- qtbase/src/corelib/text/qbytearraymatcher.h.vanilla	2021-09-17 13:14:20.790444350 +0200
+++ qtbase/src/corelib/text/qbytearraymatcher.h	2021-09-17 13:15:27.717441682 +0200
@@ -42,6 +42,8 @@
 
 #include <QtCore/qbytearray.h>
 
+#include <limits>
+
 QT_BEGIN_NAMESPACE
 
 
